src/main/java/com/google/api/codegen/viewmodel/RetryPairDefinitionView.java

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.viewmodel;
+
+import com.google.api.gax.core.RetrySettings;
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableSet;
+
+@AutoValue
+public abstract class RetryPairDefinitionView {
+
+  public abstract Name name();
+
+  public abstract ImmutableSet<String> retryCodes();
+
+  public abstract RetrySettings params();
+
+  public static Builder newBuilder() {
+    return new AutoValue_RetryPairDefinitionView.Builder();
+  }
+
+  @AutoValue.Builder
+  public static abstract class Builder {
+
+    public abstract Builder name(Name val);
+
+    public abstract Builder retryCodes(ImmutableSet<String> val);
+
+    public abstract Builder params(RetrySettings val);
+
+    public abstract RetryPairDefinitionView build();
+  }
+
+  @AutoValue
+  public static abstract class Name implements Comparable<Name> {
```

Comment:
Hmm! I seem to be missing a bunch of your comments since I thought all comments come after the "garrettjonesgoogle reviewed x hours ago" line.

Fixed. Though, I thought that this would be "obvious" from the context. RetryConfigDefinitionView has a Name. The Name has two components. Surely, the components are the names of something in the retry config?